ci/set_numpy_pandas_integration.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
Please add `set -xeo pipefail` after the shebang.

Reply:
done